### DISCUSSION TITLE
Suggestion neural-network architecture with small parameter size

### DISCUSSION THREAD

**Discussion by davidlin409** [k2-fsa/icefall#814] [Q&A]
In current icefall architecture, number of parameters for different architectures are usually large than 1 million parameters. For example:
- TDNN-LSTM: 1-million parameters
- Conformer: 10-million parameters

If somehow in some very small device, which only allows parameter size of ~ 80K parameter, the choice of neural-network architecture seems to be very restricted. 

Currently what I can think of is only to use TDNN-F from old Kaldi. Is there any more advanced method which also support small parameter size?

On the other hand, for very small device, is BPE model possible? 
 - If there are too many output in neural-network, parameter size will also be too much.

**Comment by Cirediallo** [comment]
Hello,
the subject is closed but can you point out where I can find the receipes architecture details(like how many LSTM layers was used, ...) ?

**Comment by csukuangfj** [comment]
> where I can find the receipes architecture details(like how many LSTM layers was used, ...)

Please have a look at the RESULTS.md in each recipe. You can find the training commands for each pre-trained model.
